.github/workflows/new-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: new-pr-opened
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
Should we do this also in reopen?

Reply:
I'd keep the current label state as the test might have already ran successfully, no?